server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -87,9 +87,10 @@ export function create(
         async (request, response) => {
             const tenantId = getParam(request.params, "tenantId");
             const blobData = request.body as IBlobData;
-            const historian = config.get("worker:blobStorageUrl") as string;
+            // TODO: why is this contacting external blob storage?
```

Comment:
Good question... I am not familiar with this route: ""/:tenantId/:id/blobs". Do we use it anywhere?

Reply:
Not that I know of. Likely legacy code from some long ago experiment with Blob API?

Reply:
Something for us to look at later then 😆